config/routes.rb

```diff
@@ -1,5 +1,11 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :users, only: [:new, :create]
 
-  resources :posts
```

Comment:
It looks like this removes the posts functionality, is there a way you can add users without removing the posts resource?